src/main/java/nl/ordina/jobcrawler/controller/JobcrawlerController.java

```diff
@@ -1,73 +1,95 @@
 package nl.ordina.jobcrawler.controller;
 
+
```

Comment:
Perhaps it is better to have a separate controller for the database tasks. The controller we have now in the jobcrawler project is meant to serve the front end.

Reply:
To be discussed in the meeting

Reply:
done